format_spec/array_schema.md

```diff
@@ -127,6 +103,19 @@ The dimension label has internal format:
 | Label datatype             | `uint8_t`  | The datatype of the label data |
 | Label cell_val_num         | `uint32_t` | The number of values per cell of the label data. For variable-length labels, this is `std::numeric_limits<uint32_t>::max()` |
 | Label domain size          | `uint64_t` | The size of the label domain |
-| Label domain start size    | `uint64_t` | The size of the first value of the domain for variable-lenght datatypes. For fixed-lenght labels, this is 0|
+| Label domain start size    | `uint64_t` | The size of the first value of the domain for variable-length datatypes. For fixed-length labels, this is 0|
 | Label domain data          | `uint8_t[]`| Byte array of length equal to domain size above, storing the min, max values of the dimension |
 | Is external                | `uint8_t`     | If the URI is not stored as part of this array |
+
+## Current Domain
+
+If a current domain is empty, only the version number and the empty flag are serialized to storage.
+
+The current domain format is versioned separately from arrays. The current version is `1`.
+
+| **Field** | **Type** | **Description** |
+| :--- | :--- | :--- |
+| Version number | `uint32_t` | Current domain version number |
+| Empty | `uint8_t` | Whether the current domain has a representation(e.g. NDRectangle) set or not |
```

Comment:
```suggestion
| Empty | `uint8_t` | Whether the current domain has a representation (e.g., NDRectangle) set |
```

The `or not` is implied by the use of `whether`.

Reply:
Done.